api/v1alpha1/statuschecktemplate_types.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package v1alpha1
+
+const (
+	TemplateLabelKey          = "chaos-mesh.org/template-type"
+	ManagedByLabelKey         = "app.kubernetes.io/managed-by"
+	PrefixStatusCheckTemplate = "template-status-check"
+)
```

Comment:
please add comments for the usage of these labels.

Reply:
Updated.